projects/wallet-template/src/containers/WalletPopup/pages/Accounts/AddAccount/index.tsx

```diff
@@ -66,7 +66,15 @@ export const AddAccount = () => {
       await rpc.client.insertKeyset(data.keysetName, {
         scheme: "Sr25519",
         derivationPaths,
+        createdAt: Date.now(),
       })
+      await rpc.client.setPrimaryKeysetName(data.keysetName)
+      await mutate(
+        (key) =>
+          Array.isArray(key) &&
+          typeof key[0] === "string" &&
+          key[0].startsWith("/rpc/keysets"),
+      )
```

Comment:
@kratico i think this is ok?

https://swr.vercel.app/docs/mutation#mutate-multiple-items

Reply:
same as before, we want to avoid hardcoding keys so it's better to use the bounded mutate